pxr/base/tf/testenv/error.cpp

```diff
@@ -29,7 +29,8 @@
 
 #include "pxr/base/arch/functionLite.h"
 
-#include <tbb/tbb_thread.h>
+//#include <tbb/tbb_thread.h>
```

Comment:
Please don't leave commented out code that has been deprecated. However, again we need to make sure these changes are backwards compatible.